src/seeders/20191209122959-users.js

```diff
@@ -0,0 +1,44 @@
+import faker from 'faker';
+import authManager from '../helpers/jwt';
+
+const { hashPassword } = authManager;
+
+module.exports = {
+  up: (queryInterface) => queryInterface.bulkInsert('Users', [{
+    firstName: faker.name.firstName(),
+    lastName: faker.name.lastName(),
+    userName: faker.internet.userName(),
```

Comment:
Username is not in the new model that Temitayo suggested. You can remove it. As I did too. Thx